server/src/objects/github.rs

```diff
@@ -0,0 +1,92 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2024 Aliaksei Bialiauski
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+use crate::objects::user::User;
+use std::collections::HashMap;
+use uuid::Uuid;
+
+/// GitHub.
+#[derive(Clone)]
+pub struct GitHub {
+    /// GitHub ID.
+    pub id: Uuid,
+    /// GitHub URL.
+    pub name: String,
+    /// Users inside.
+    pub users: HashMap<String, User>,
```

Comment:
`UUID` maybe?
```suggestion
    pub users: HashMap<UUID, User>,
```

Reply:
@l3r8yJ hm, UUID is not really handy here, since we want only to query users by their names (which is already unique). WDYT?